tests/contrib/django/app/settings_untraced.py

```diff
@@ -97,10 +97,21 @@
     'django.contrib.auth',
     'django.contrib.contenttypes',
     'django.contrib.sessions',
+
```

Comment:
Why is it not needed to have the `if django.VERSION >= (1, 10):` check here too?

Reply:
Yep !